src/scss/components/_rate.scss

```diff
@@ -0,0 +1,73 @@
+$rate-color: $grey-lighter !default;
+$rate-color-active: darken($warning, 5%) !default;
+$rate-space: 5px !default;
```

Comment:
I think we should try to use `em` or `rem` instead of px (there are other places too)

Reply:
Maybe something like `0.25rem` for the `rate-space`? It would be the same item spacing as Bulma pagination (`$pagination-item-margin`).